config/initializers/003-sql_builder.rb

```diff
@@ -1,3 +0,0 @@
-# frozen_string_literal: true
-
-require 'sql_builder'
```

Comment:
No need for this anymore as it’s been deprecated for a while (and should be dropped with the 2.9 version).